src/GitHub.VisualStudio/Styles/GitHubComboBox.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI">
+    
+    <Style x:Key="GitHubComboBoxBorder" TargetType="{x:Type Border}">
+        <Setter Property="Margin" Value="10" />
+        <Setter Property="BorderThickness" Value="1" />
+        <Setter Property="BorderBrush" Value="{DynamicResource GitHubComboBoxBorderBrush}" />
+        <Setter Property="Effect">
+            <Setter.Value>
+                <DropShadowEffect Direction="315" ShadowDepth="5" BlurRadius="5" Opacity="0.25" />
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Key="GitHubComboBoxContainer" TargetType="{x:Type Grid}">
+        <Setter Property="Background" Value="{DynamicResource GitHubComboBoxBackgroundBrush}" />
+
+        <Style.Resources>
+            <Style TargetType="{x:Type TextBlock}" BasedOn="{StaticResource {x:Type TextBlock}}">
+                <Setter Property="Foreground" Value="{DynamicResource GitHubVsToolWindowText}" />
+            </Style>
+
+            <Style TargetType="{x:Type ui:OcticonImage}" BasedOn="{StaticResource {x:Type ui:OcticonImage}}">
```

Comment:
Hmmm, the `xmlns:ui` declaration is correct, the `OcticonImage` type is in the `GitHub.UI` namespace as declared here: https://github.com/github/VisualStudio/blob/master/src/GitHub.UI/Controls/Octicons/OcticonImage.cs#L5.
It might need the GitHubUI SharedDictionary.xaml file where the `OcticonImage.xaml` is included from (where the style is eventually declared).

Try adding to the top:

```
<ResourceDictionary.MergedDictionaries>
   <cache:SharedDictionaryManager Source="pack://application:,,,/GitHub.UI;component/SharedDictionary.xaml" />
</ResourceDictionary.MergedDictionaries>
```